test/Spec.hs

```diff
@@ -1,19 +1,22 @@
 module Main (main) where
 
+import Spec.BotPlutusInterface.AdjustUnbalanced qualified
 import Spec.BotPlutusInterface.Balance qualified
 import Spec.BotPlutusInterface.CoinSelection qualified
-import Spec.BotPlutusInterface.Collateral qualified
 import Spec.BotPlutusInterface.Contract qualified
 import Spec.BotPlutusInterface.ContractStats qualified
 import Spec.BotPlutusInterface.Server qualified
 import Spec.BotPlutusInterface.TxStatusChange qualified
 import Spec.BotPlutusInterface.UtxoParser qualified
+import System.IO
 import Test.Tasty (TestTree, defaultMain, testGroup)
-import Prelude
 
 -- | @since 0.1
 main :: IO ()
-main = defaultMain tests
+main = do
+  hSetBuffering stdin LineBuffering
```

Comment:
is this needed? for terminal outputs its the default either way

Reply:
Yes, `LineBuffering` is default kind of buffering.

cc: @mikekeke

Reply:
Fixed!